terraform-continuous-deployment/terraform-continuous-deployment.yaml

```diff
@@ -0,0 +1,49 @@
+apiVersion: v1
+description: Automatically run a Terraform plan when a GitHub PR is merged
+
+triggers:
+- name: github-pr-merge
+  source:
+    type: webhook
+    image: relaysh/github-trigger-pull-request-merged:latest
+  binding:
+    parameters:
+      repository: !Data repository
+      branch: !Data branch
+  when:
+  - !Fn.equals [!Data branch, "master"]
+
+parameters:
+  repository:
+    description: The name of the repository that had a PR merged
+  branch:
+    description: The branch that the PR was pulled into
+    default: master
+  terraformWorkspace:
+    description: The Terraform workspace to use
+    default: default
+  terraformPlanDir:
+    description: The directory of the Terraform plan
```

Comment:
Since this doesn't have a default, won't it need to be set in the trigger binding?